content/zh/docs/unified_scheduling.md

```diff
@@ -0,0 +1,263 @@
++++
+title = "统一调度"
+date = 2024-12-30
+lastmod = 2024-12-30
+draft = false
+toc = true
+type = "docs"
+
+[menu.docs]
+  parent = "features"
+  weight = 3
++++
+
+## 概述
+
+Volcano 作为业界领先的云原生批处理系统调度器，通过统一的调度系统实现了对所有工作负载类型的支持：
+
+- 强大的批处理调度能力：通过 VcJob 完美支持 TensorFlow、PyTorch、Ray、Spark 等主流 AI 和大数据框架
+- 完整的 Kubernetes 工作负载支持：可直接调度 Deployment、StatefulSet、Job、DaemonSet 等原生工作负载
+
+这种统一调度能力让用户能够使用单一调度器来管理所有类型的工作负载，极大简化了集群管理复杂度。
+
+## 兼容Kubernetes调度能力
+
+### 1. 节点过滤机制 (Predicates)
+Volcano完整实现了Kube-Scheduler中的核心节点过滤机制,包括：
```

Comment:
We should emphasize that the k8s related features are hosted by predict and nodeorder seperately.
And remove (Predicates) and (NodeOrder).

Reply:
updated, now use predicates plugin and nodeorder plugin